notebooks/deepdive/custom/ucaip_customjob.ipynb

```diff
@@ -0,0 +1,1857 @@
+{
```

Comment:
<p>FYI - The Stackdriver product name has changed to a suite of products called "Operations" (https://cloud.google.com/products/operations). The product suite includes: Cloud Logging, Cloud Monitoring, Cloud Trace, Cloud Debugger, and Cloud Profiler.</p><p> </p><p>Here, you'd use "Cloud Logging" - please make this replacement in all affected notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>